src/Avalonia.X11/X11Structs.cs

```diff
@@ -772,7 +773,8 @@ internal enum XEventName {
 		ClientMessage		= 33,
 		MappingNotify		= 34,
         GenericEvent = 35,
-		LASTEvent
+        XShmCompletionEvent = 65,
```

Comment:
It's not a constant, you are supposed to call `XShmGetEventBase`

Reply:
Thank you @kekekeks .

From https://www.[x.org/releases/X11R7.5/doc/Xext/mit-shm.html](https://www.x.org/releases/X11R7.5/doc/Xext/mit-shm.html)

The XShmCompletionEvent  event type value that will be used can be determined at run time with a line of the form:

    int CompletionType = XShmGetEventBase (display) + ShmCompletion;